src/Test/UnitTests/attributes.jl

```diff
@@ -31,6 +31,27 @@ function silent(model::MOI.ModelLike, config::TestConfig)
 end
 unittests["silent"] = silent
 
+"""
+    timelimit(model::MOI.ModelLike, config::TestConfig)
+
+Test that the [`MOI.TimeLimit`](@ref) attribute is implemented for `model`.
+"""
+function timelimit(model::MOI.ModelLike, config::TestConfig)
+    if config.solve
+        @test MOI.supports(model, MOI.TimeLimit())
+        # Get the current value to restore it at the end of the test
+        value = MOI.get(model, MOI.TimeLimit())
+        MOI.set(model, MOI.TimeLimit(), value + 1)
```

Comment:
These tests with `value + 1` seem overly complex. Why not do:
```julia
value = MOI.get(model, MOI.TimeLimit())

MOI.set(model, MOI.TimeLimit(), 15.0)
@test 15.0 ≈ MOI.get(model, MOI.TimeLimit())

MOI.set(model, MOI.TimeLimit() value)
@test value == MOI.get(model, MOI.TimeLimit()) # Equality should hold.
```

Reply:
I'm changing that part to check 2 values:
```Julia
MOI.set(model, MOI.TimeLimit(), 0.0)
@test isapprox(MOI.get(model, MOI.TimeLimit()), 0.0)
# Check that `set` does not just set TimeLimit to `0.0`
MOI.set(model, MOI.TimeLimit(), 15.0)
@test isapprox(MOI.get(model, MOI.TimeLimit()), 15.0)
```

Then for `value` I was thinking of:
```Julia
MOI.set(model, MOI.TimeLimit(), value)
if value !== nothing
    @test isapprox(value, MOI.get(model, MOI.TimeLimit()))
else
    @test value === MOI.get(model, MOI.TimeLimit())
end
```
but if equality should hold is enough I can do that instead.

Reply:
> equality should hold is enough I can do that instead

I believe it's safe to assume that equality will hold for a time limit that the solver already had set. There shouldn't be loss in the process.